src/richie/apps/courses/models/blog.py

```diff
@@ -24,6 +24,7 @@ class Meta:
         db_table = "richie_blog_post"
         ordering = ["-pk"]
         verbose_name = _("blog post")
+        verbose_name_plural = _("blog posts")
```

Comment:
There are no `.po` files committed.

Reply:
I tried to generate them but they include a lot of fuzzy that we don't want to commit... the best is to let the CI generate them and send them to CircleCI. We will then translate everything in CircleCI and update the .po files by pulling from CircleCI. It is also better because we commit string directly with their translations.